adapter/src/dummy.rs

```diff
@@ -34,24 +34,29 @@ impl Adapter for DummyAdapter {
         Ok(())
     }
 
-    fn whoami(&self) -> String {
+    fn whoami(&self) -> ValidatorId {
```

Comment:
Why not a reference and wherever we need owned, we can `clone()` or `to_owned()`?